main.js

```diff
@@ -19,15 +19,6 @@ if (config.jars) {
   jars = jars.concat(config.jars.split(":"));
 }
 
-if (config.pushConn && config.pushMidlet) {
-  MIDP.ConnectionRegistry.addConnection({
-    connection: config.pushConn,
-    midlet: config.pushMidlet,
-    filter: "*",
-    suiteId: "1"
-  });
-}
-
```

Comment:
Nit: there are also references to pushConn and pushMidlet in config/urlparams.js that should get removed.